v2/thirdparty/common-customizations/saml/with-boxyhq/integration-steps.mdx

```diff
@@ -276,7 +279,7 @@ func main() {
 	// highlight-start
 	resp, err := multitenancy.CreateOrUpdateThirdPartyConfig(tenantId, tpmodels.ProviderConfig{
 		ThirdPartyId: "boxy-saml",
-		Name:         "SAML Login",
+		Name:         "<provider-name>",
```

Comment:
you need to specify the list of provider names that we support! How will the user know otherwise?

Reply:
Updated